app/components/viral/sortable_list/list_component.html.erb

```diff
@@ -0,0 +1,25 @@
+<div class="<%= @system_arguments[:container_classes] %>">
+  <%= title %>
+  <ul
+    id="<%= id %>"
+    data-controller="sortable-list"
+    data-sortable-list-group-name-value="<%= group %>"
+    class="<%= @system_arguments[:list_classes] %>"
+    >
+    <% unless list_items.empty? %>
```

Comment:
We would want the possibility of an empty list, much like the lists in this PR where the selected list starts empty. My unless statement was to avoid doing an `.each` loop on an empty list, but it looks like it works fine on an empty array, so I could take out the `unless` statement.

Reply:
Ended up removing the `unless` statement 287ef3ce83906309846f4ec9d91b4430c3a826fb